test/foundry/deploy/12_holograph_genesis_tests.t.sol

```diff
@@ -71,7 +71,6 @@ contract HolographGenesisTests is Test {
   function testSuccessfulyDeployed() public {
     uint256 chainId = block.chainid;
     bytes memory mockBytecode = vm.getCode("Mock.sol:Mock");
-    bytes memory initCode = abi.encode(0x0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcd);
     vm.prank(deployerGenesisLocal);
     holographGenesis.deploy(
       chainId,
```

Comment:
where is the init code? used in deploy()

Reply:
initCode() was defined at the beginning of the contract since it is used by several tests.